### PR TITLE
[3]docs: adds integrity check section

### LINKED ISSUE #3974: [3]documentation: integrity checks and nginx
** Read before you start ** https://www.divio.com/blog/documentation/ (Explanation section)
Check the parent task: https://github.com/inveniosoftware/invenio/issues/3882 and comment for structure of the documentation, and write documentation for sections listed below:

	- Configuring integrity checks
		- setting up your low priority celery queue
	- Configuring nginx to allow for large file uploads

### PR DESCRIPTION
**Pull request by roksys**
* Closes #3974

### CODE REVIEW

File: docs/managing-files/integrity.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Integrity checks
+================
+
+`Invenio-Files-REST <https://invenio-files-rest.readthedocs.io/>`_ provides
+Celery task-based mechanism for ensuring files integrity. It consists of two
+dedicated functions (Celery tasks) in :py:mod:`invenio_files_rest.tasks`.
+
+Main task ``invenio_files_rest.tasks.schedule_checksum_verification`` is to be
```
Comment (topless) [inline docs/managing-files/integrity.rst]: is to be -> will be called periodically ...

File: docs/managing-files/integrity.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+periodically called through `celery beat <https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#id1>`_
+and is responsible for querying files and scheduling a subtask
```
Comment (topless) [inline docs/managing-files/integrity.rst]: Not sure we have to be so thorough about how celery beat works. We can simplify it , something like:

will be called periodically and will run integrity checks ``invenio_files_rest.tasks.verify_checksum`` on the files.

File: docs/managing-files/integrity.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#starting-the-scheduler>`_:
+
+.. code-block:: console
+
+    $ celery -A proj beat
+
+Once celery beat service is running, you need to define a task for it in your
+application's ``config.py``.
+
+.. code-block:: python
```
Comment (topless) [inline docs/managing-files/integrity.rst]: https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

Check the fifth bullet

File: docs/managing-files/integrity.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+``invenio_files_rest.tasks.verify_checksum`` for each file which integrity needs
+to be checked.
+
+Configuration
+-------------
+
+First, you need to `start the celery beat service
+<https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#starting-the-scheduler>`_:
+
+.. code-block:: console
```
Comment (topless) [inline docs/managing-files/integrity.rst]: I think I left a comment before for this one, you can be advised https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

`.. console::`

**Comment by ntarocco** [comment]
Will take all comments into account.

**Pull request merged**